fabric-renderer-indigo/src/main/java/net/fabricmc/indigo/renderer/mesh/MutableQuadViewImpl.java

```diff
@@ -138,13 +124,27 @@ public MutableQuadViewImpl pos(int vertexIndex, float x, float y, float z) {
     @Override
     public MutableQuadViewImpl normal(int vertexIndex, float x, float y, float z) {
         normalFlags |= (1 << vertexIndex);
-        data[baseIndex + VERTEX_START_OFFSET + VANILLA_STRIDE + vertexIndex] = NormalHelper.packNormal(x, y, z, 0);
+        data[baseIndex + vertexIndex * 8 + 7 + VERTEX_START_OFFSET] = NormalHelper.packNormal(x, y, z, 0);
```

Comment:
Yes, the packing has changed because normals are now part of the vanilla format and come at the end of the vertex stride.  Previously they we tagged on after the vanilla data.